src/transcriptProcessing/updateOutputTimes.ts

```diff
@@ -43,10 +43,30 @@ export const calculateTime: (
   return roundToMs(prevWord.outputStartTime + bufferedWordDuration(prevWord));
 };
 
+const updateOutputVideoDuration: (words: Word[]) => number = (words) => {
+  const currentTranscriptWords = words.filter((word) => word.deleted === false);
+  const lastWord = currentTranscriptWords.pop();
```

Comment:
`pop` mutates, which we want to avoid in redux. Instead check the length and if length > 0 then read the final word

Reply:
Resolving as we had a convo about this